update colors and whats new page (#1445)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

## Febuary 22-26, 2021

### New Spinner Component on Grommet.

The new Spinner component has been released on grommet stable [branch](https://github.com/grommet/grommet/pull/4981).

## Febuary 15-19, 2021

### New FileInput Component
- Check out the new [FileInput](https://v2.grommet.io/fileinput) Component on the grommet stable branch.

### New Pagination Component 

> - The new [Pagination Component](https://v2.grommet.io/pagination) is ready to be used!

Reviewer: ```suggestion
- Check out the new [Pagination Component](https://v2.grommet.io/pagination) on the grommet stable branch.
```

### Re-align button sizes

> - Adjusted Button sizes to use [primary](https://github.com/grommet/hpe-design-system/pull/1433).

Reviewer: I don't think we need to share this one, it's just regarding a tiny site example but not really a change someone would see in their app

Author: okay that's fine I wasn't sure where we draw the line

Reviewer: No worries! I think in general we are thinking about What's New as a sort of "weekly release notes" in a sense, so any change that will affect how a team's app might look/new guidance on how things should behave

### Layer 

- Aligned [layer component](https://github.com/grommet/hpe-design-system/pull/1434) to match the figma files. 

## Febuary 8-12, 2021

### Homepage 

> - Completed a fresh new look for our new HPE design system [home page](https://github.com/grommet/hpe-design-system/pull/1408).

Reviewer: ```suggestion
- Completed a fresh new look for our new HPE Design System [home page](https://github.com/grommet/hpe-design-system/pull/1408).
```

- [Figma](https://www.figma.com/file/CWB0xMmfVx6cT0ugzluySl/home-page?node-id=87%3A379) is updated as well 

> ### Added Wizard themplate 

Reviewer: ```suggestion
### Updated Wizard template 
```
